components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.common/src/main/java/org/wso2/carbon/identity/api/server/idp/common/Constants.java

```diff
@@ -146,6 +146,18 @@ public enum ErrorMessage {
         ERROR_CODE_ERROR_LISTING_TRUSTED_TOKEN_ISSUERS("60021",
                 "Unable to list existing trusted token issuers.",
                 "Server encountered an error while listing the trusted token issuers."),
+        ERROR_CODE_ENDPOINT_PROVIDED_FOR_SYSTEM_AUTH("60039", "No endpoint configuration is allowed " +
+                "for system defined authenticators.", "No endpoint configuration must be " +
+                "provided for the system defined federated authenticators %s."),
+        ERROR_CODE_PROPERTIES_PROVIDED_FOR_USER_AUTH("60040", "No properties are allowed for " +
+                "user defined authenticators.", "No properties must be provided for the user defined " +
+                "federated authenticators %s."),
```

Comment:
Can we simplify the error message of these two cases. It's almost similar to a description